### PR TITLE
browser(symphony) - Oct192022

### PR DESCRIPTION
**Pull request by a-sassman**
This symphony PR includes image updates on 11 browser pages, which is a part of the browser symphony ticket https://issues.newrelic.com/browse/NR-51378 .

### CODE REVIEW

File: src/content/docs/browser/new-relic-browser/browser-pro-features/browser-data-distributed-tracing.mdx [hunk 1]
```diff
@@ -10,7 +10,9 @@ redirects:
   - /docs/distributed-tracing/ui-data/browser-data/
 ---
 
-import distributedTracingBrowserSpanIcon from 'images/distributed-tracing-browser-span-icon.png'
+import browserBrowserIcon from 'images/browser_icon_browser-icon.png'
+
+import browserErrorProfile from 'images/browser_screenshot-crop_error-profile.png'
```
Comment (paperclypse) [inline src/content/docs/browser/new-relic-browser/browser-pro-features/browser-data-distributed-tracing.mdx:15]: @a-sassman it looks like this import statement doesn't have a matching image tag in the doc itself

File: src/content/docs/browser/new-relic-browser/browser-pro-features/browser-error-profiles-find-error-causes.mdx [hunk 2]
```diff
@@ -29,7 +29,7 @@ The screenshot below shows an example of an error profile.
 <img
   title="jserror_agentos.png"
   alt="jserror_agentos.png"
-  src={jserrorAgentos}
+  src={browserErrorProfileExample1}
```
Comment (paperclypse) [inline src/content/docs/browser/new-relic-browser/browser-pro-features/browser-error-profiles-find-error-causes.mdx:32]: @a-sassman it looks like this should be `browserErrorProfileExample21`

File: src/content/docs/browser/new-relic-browser/browser-pro-features/filterable-geography-webpage-metrics-location.mdx [hunk 3]
```diff
@@ -62,7 +54,7 @@ To view or sort your browser monitoring performance data by location:
 <img
   title="Filterable geography charts UI"
   alt="Screenshot of Filterable geography charts UI in New Relic"
-  src={BrowserFilterGeos2}
+  src={browserGeoData}
```
Comment (paperclypse) [inline src/content/docs/browser/new-relic-browser/browser-pro-features/filterable-geography-webpage-metrics-location.mdx:57]: @a-sassman looks like you're missing an import statement for this one.

File: src/content/docs/browser/new-relic-browser/browser-pro-features/browser-error-profiles-find-error-causes.mdx (hunk 2 above)
Comment (a-sassman) [reply src/content/docs/browser/new-relic-browser/browser-pro-features/browser-error-profiles-find-error-causes.mdx:32]: I fixed this with the latest commit.

File: src/content/docs/browser/new-relic-browser/browser-pro-features/filterable-geography-webpage-metrics-location.mdx (hunk 3 above)
Comment (a-sassman) [reply src/content/docs/browser/new-relic-browser/browser-pro-features/filterable-geography-webpage-metrics-location.mdx]: I fixed this with the latest commit.

File: src/content/docs/browser/browser-monitoring/browser-pro-features/session-traces-explore-webpages-life-cycle.mdx [hunk 4]
```diff
@@ -39,11 +37,11 @@ Our browser monitoring's **Session traces** UI provides a detailed timeline of t
 <img
   title="Session trace error details"
   alt="A screenshot of error details in a browser session trace"
-  src={browserSessiontraceError}
+  src={browserBrowserSessionerror.png}
```
[the remark this answers is not in the document]
Comment (a-sassman) [reply src/content/docs/browser/browser-monitoring/browser-pro-features/session-traces-explore-webpages-life-cycle.mdx:40]: Resolved in latest commit

**Pull request merged**